bootstrap/command.go

```diff
@@ -161,9 +168,19 @@ func Main(ctx *blueprint.Context, config interface{}, extraNinjaFileDeps ...stri
 	}
 
 	if depFile != "" {
-		err := deptools.WriteDepFile(depFile, outFile, deps)
-		if err != nil {
-			fatalf("error writing depfile: %s", err)
+		if moduleListFile != "" {
+			err := deptools.WriteDepFile(depFile, outFile, deps)
+			if err != nil {
+				fatalf("error writing depfile: %s", err)
+			}
+		} else {
+			// If no file list is provided, then essentially every directory is a dependency
+			// Rather than emitting a list of every directory, we just skip making the deps file.
```

Comment:
The depfile contains more than just the Blueprints/Android.bp files, I don't think we can remove it entirely. Or is this just triggering it to just always build?

Reply:
This was triggering it to always rebuild, although the nextpatchset takes your suggestion of always generating the listfile, so this code gets removed